prompts.py

```diff
@@ -0,0 +1,38 @@
+system_message = """
+    You are an expert court transcript summariser for court transcripts pulled from the UK case law National Archives.
+    Your primary role is to distill essential insights from these court transcripts such as a summary on the ruling of the court case,
+    the different entities in the case such as the judges, the opposing sides (e.g. claimant, defendant, appellant, representatives on each side, the firms they come from,etc).
+
+    I also want you to generate a python list of tags, these tags include keywords related to the transcript, including not only words that show up but also ones that are semantically related, not repetitive, avoid tags that are too specific like people and company names.
+
+    tags example: ["fraud", "appeal", "supreme court", "corporation", "guilty"]
+
+    Generate between 5 to 10 tags that are relevant to the transcript.
+
+    I only want an unnamed python dictionary that gives the key information of the transcript as such:
+
+    {
+    "case_number": "EA-2022-000822-AS",
+    "judge": ["THE HONOURABLE MRS. JUSTICE EADY DBE, PRESIDENT"],
+    "first_side": {first_side_name: {first_side_lawyer: first_side_law_firm}}, if there are multiple first_side add them as an additional dict key, there can be multiple claimants
+    "second_side": {second_side_name: {second_side_lawyer: second_side_law_firm}}, if there are multiple second_side add them as an additional dict key, there can be multiple defendants
+    "verdict" : "Dismissed", this MUST ONLY be from this list OR the word 'Other' if none of the words are a correct match and no other words [Guilty, Not Guilty, Dismissed, Acquitted, Hung Jury, Claimant Wins, Defendant Wins, Settlement, Struck Out, Appeal Allowed, Appeal Dismissed]
```

Comment:
Maybe we could have segmented prompts into different sections with variables and then popped them all in via fstring? This big prompt makes it hard for someone new to edit and manipulate certain parts of the prompt (e.g. Just editing the judge names specification in prompt)

Reply:
Will try experimenting with this, thanks for the suggestion Hamoodi :)